google-cloud-speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -309,15 +331,18 @@ def long_running_recognize \
           # @example
           #   require "google/cloud/speech/v1"
           #
-          #   speech_client = Google::Cloud::Speech::V1::SpeechClient.new
+          #   speech_client = Google::Cloud::Speech::V1.new
           #   request = {}
           #   requests = [request]
           #   speech_client.streaming_recognize(requests).each do |element|
           #     # Process element.
           #   end
 
           def streaming_recognize reqs, options: nil
-            @streaming_recognize.call(reqs, options)
+            request_protos = reqs.lazy.map do |req|
```

Comment:
I think this has broken the streaming acceptance tests, although I haven't taken a closer look. Any idea why?

FYI @landrito